stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -1081,6 +1166,9 @@ private static StripeResponse getStripeResponse(
             Map<String, Object> params,
             RequestOptions options)
             throws InvalidRequestException, APIConnectionException {
+        if (mockGetStripeResponse != null){
```

Comment:
add space before `{`

Reply:
Interesting, it did not get caught by the linter, changed